application/application.go

```diff
@@ -12,8 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package collector handles the command-line, configuration, and runs the OC collector.
-package collector
+// Package application handles the command-line, configuration, and runs the
+// OpenTelemetry Service.
+package application
```

Comment:
Maybe better call this `service` package?

Reply:
Done